### PR TITLE
Install the correct systemd unit file on older RPM systems.

### PR DESCRIPTION
**Pull request by Ferroin**
##### Summary

CentOS 7 and derivatives need the pre-v235 systemd unit files, not the modern ones.

##### Test Plan

Packages built from this PR install the correct unit files, and the agent starts up correctly.

##### Additional Information

Fixes: #15236

### CODE REVIEW

File: netdata.spec.in [hunk 1]
```diff
@@ -342,8 +342,13 @@ install -m 750 -p packaging/installer/netdata-uninstaller.sh \
 
 # ###########################################################
 # Install netdata service
+
 install -m 755 -d "${RPM_BUILD_ROOT}%{_unitdir}"
+%if 0%{?centos_ver} != 7 && 0%{?amzn} != 2
```
Comment (ilyam8) [inline netdata.spec.in:347]: Should it be `amazon_linux`? I CTRL-F and found only `0%{?amazon_linux}` in this file.

Comment (Ferroin) [reply netdata.spec.in:347]: Quite so, I forgot that they changed their macro names between versions.

**Review by ilyam8 — APPROVED** [review]

**Comment by tkatsoulas** [comment]
I am seeing this error in the systemctl

```
Jun 22 18:51:33 easy.box ebpf.plugin[10721]: set name of thread 10764 to EBPF OOMKILL
Jun 22 18:51:33 easy.box ebpf.plugin[10721]: ERROR: loading BPF object file failed /usr/libexec/netdata/plugins.d/ebpf.d/rnetdata_ebpf_process.3.10.rhf.o
Jun 22 18:51:33 easy.box ebpf.plugin[10721]: thread with task id 10760 finished
Jun 22 18:51:33 easy.box ebpf.plugin[10721]: thread with task id 10762 finished
Jun 22 18:51:33 easy.box ebpf.plugin[10721]: thread with task id 10764 finished
Jun 22 18:51:33 easy.box ebpf.plugin[10721]: ERROR: loading BPF object file failed /usr/libexec/netdata/plugins.d/ebpf.d/rnetdata_ebpf_cachestat.3.10.rhf.o
Jun 22 18:51:33 easy.box ebpf.plugin[10721]: Cannot open or load BPF file for thread cachestat
```

**Review by tkatsoulas — APPROVED** [review]

**Comment by tkatsoulas** [comment]
Capabilities are also good, systemd service file doesn't include them though (I think it is expected).

**Pull request merged**